src/index.html

```diff
@@ -5,8 +5,387 @@
+                >
+                  Tablets
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Photo
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__list"
+                  href="#"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </header>
+      </div>
+
+
+      <main class="main catalog__main">
+        <div class="first-four">
+          <div class="card" data-qa="card">
+            <img
+              class="card__image"
+              src="./images/imac.jpeg"
+              alt="imac"
+            >
+
+            <p class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__rate rate">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="rate__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-title">Price:</p>
+              <p class="card__price-number">$2,199</p>
+            </div>
+            <a
+              class="card__link"
+              href="#"
+            >
+              <p
+                class="card__link-text"
+                data-qa="card-hover"
+              >
+                BUY
+              </p>
+            </a>
+          </div>
+
+          <div class="card"
+            >
+            <img
+              class="card__image"
+              src="./images/imac.jpeg"
+              alt="imac"
+            >
+
+            <p class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__rate rate">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="rate__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-title">Price:</p>
+              <p class="card__price-number">$2,199</p>
+            </div>
+            <a
+              class="card__link"
+              href="#"
+            >
+              <p class="card__link-text">BUY</p>
```

Comment:
do you need `p` here?
I believe you can move style from `p` to `a` and remove tag `p`
Also, don't write text in uppercase, use css for it